resources/views/checkout/steps/login.blade.php

```diff
@@ -12,17 +12,27 @@
                 v-on:input="loginInputChange"
                 required
             />
-            <x-rapidez::input
-                v-if="!emailAvailable"
-                :label="false"
-                class="mt-3"
-                name="password"
-                type="password"
-                placeholder="Password"
-                ref="password"
-                v-on:input="loginInputChange"
-                required
-            />
+            <div class="relative">
+                <x-rapidez::input
+                    v-if="!emailAvailable"
+                    :label="false"
+                    class="mt-3"
+                    name="password"
+                    type="password"
+                    placeholder="Password"
+                    ref="password"
+                    v-on:input="loginInputChange"
+                    required
+                />
+
+                <input
+                    type="checkbox"
+                    v-if="!emailAvailable"
+                    oninvalid="this.setCustomValidity('{{ __('Please log in') }}')"
+                    class="absolute h-full inset-0 opacity-0 pointer-events-none"
```

Comment:
is this the way to go? absolute inset-0 pointer events none? seems more like a quick fix rather than a proper one?

Reply:
We need some sort of extra input that the browser still renders, so that we can display the validation message. There isn't really a better method for that :)